sysbuild/bootloader/Kconfig

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config SUPPORT_BOOTLOADER
+	bool
+	default y
+
+config SUPPORT_BOOTLOADER_MCUBOOT_ZEPHYR
+	bool
+	default y
+
+choice BOOTLOADER
+	prompt "Bootloader support"
+	default BOOTLOADER_NONE
+	depends on SUPPORT_BOOTLOADER
+
+config BOOTLOADER_NONE
+	bool "None"
+	help
+	  Do not Include a bootloader in the build
+
+config BOOTLOADER_MCUBOOT
+	bool "MCUboot"
+	depends on SUPPORT_BOOTLOADER_MCUBOOT_ZEPHYR
+	help
+	  Include MCUboot (Zephyr port) as the bootloader to use
+
+endchoice
+
+if BOOTLOADER_MCUBOOT
+
+config SIGNATURE_TYPE
```

Comment:
Why this and following Kconfig properties has to be repeated in this file?
MCUboot original Kconfig already contain these definitions.

Reply:
please see a related answer here:
https://github.com/zephyrproject-rtos/zephyr/pull/40555#issuecomment-1158710808

when we have a higher level sysbuild, then there can be settings that needs to be propagated.
In this case, knowledge on how to sign images naturally belongs on the top, in order to know how to sign, and where the keys are located.

This information can then be passed down to other images, such as MCUboot, but similar information could be needed for other bootloaders.